README.md

```diff
@@ -86,6 +86,67 @@ def play(self):
 			return self.energy
 ```
 
+and add the following lines right under `time.sleep(self.latency)` on line 27:
+
+```python
+if debug:
+	next = input()
+	if next:
+		return 'Next'
+```
+
+### Edit `Map.py`
+Lastly, in `Map.py`, add:
+
+```python
+self.overlap = [str(i) for i in range(10)]
+```
+
+on line 19, right below:
+
+```python
+self.botPosition = self.checkpoint[:] #bot always starts at the checkpoint` on line 18.
+```
+
+Then, change the signature of the `moveRobot()` method on line 58, from:
+
+```python
+def moveRobot(self, move)
+```
+
+to:
+
+```python
+def moveRobot(self, move, debug=False)
+```
+
+and change the lines after:
+
+```python
+elif move == 'right':
+	self.botPosition[1] = self.botPosition[1]+1
+	if self.map[self.botPosition[0]][self.botPosition[1]] == '@':
+		self.remainingStains -=1
+```
+
+to the ones below, but make sure to stay within the `moveRobot()` method:
+
+```python
+self.currentSign = self.map[self.botPosition[0]][self.botPosition[1]]
+self.map[self.botPosition[0]][self.botPosition[1]] = 'B'
```

Comment:
I tested the scripts using RandomBot, and it's a bit challenging to distinguish my robot ('B') from several '8's. Therefore, I suggest we continue using '\xC6(Æ)' to denote the robot.

```python
self.map[self.botPosition[0]][self.botPosition[1]] = '\xC6'
```

Reply:
Yeah that's completely fine, I changed it to 'B' because I accidentally deleted '\xC6(Æ)' and couldn't find it anymore haha